halo2_proofs/src/dev.rs

```diff
@@ -979,7 +978,7 @@ mod tests {
                         config.q.enable(&mut region, 1)?;
 
                         // Assign a = 0.
-                        region.assign_advice(|| "a", config.a, 0, || Value::known(Fp::ZERO))?;
+                        region.assign_advice(|| "a", config.a, 0, || Value::<Fp>::ZERO)?;
```

Comment:
Similarly here we cannot avoid the type annotation because of how flexibly the `Region::assign_advice` API is (intentionally) set up, so we only save a few characters.